tests/framework/checks/skips.go

```diff
@@ -225,3 +227,9 @@ func SkipIfARM64(arch string, message string) {
 		ginkgo.Skip("Skip test on arm64: " + message)
 	}
 }
+
+func SkipIfMultusV3() {
+	if strings.ToLower(os.Getenv("KUBEVIRT_WITH_MULTUS_V3")) == "true" {
```

Comment:
nit: `strings.ToLower` do we really need this ?

Reply:
It won't fail the whole suite if someone set `TRUE`/`True` instead of `true`.

Reply:
Ye I understand, I just think we don't need to cover such cases
I think we don't cover those cases all over for the rest of the parameters

Reply:
From a user perspective, it's very annoying to run the network tests and have these tests skipped just because I use the exact format.

Maybe https://pkg.go.dev/strconv#ParseBool would be a better fit.

Reply:
Since [here](https://github.com/kubevirt/kubevirtci/pull/977/files#diff-592e26bb575a60fc920cb2ab9f9cb0fd10609ebd36b19c35ddd1d57ba0fc0cd3R70) check the lower case version only I agree with @oshoval.